### DISCUSSION TITLE
查看范围如何这样的效果

### DISCUSSION THREAD

**Discussion by zhiyou1201** [antvis/G2#5730] [Q&A]
查看范围如何这样的效果
![image](https://github.com/antvis/G2/assets/75651528/b50afd6a-31b0-4652-abcc-ca2259e2e0b0)

如何实现这样的效果

https://github.com/antvis/G2/assets/75651528/41d970be-c0b8-4fcd-a4af-6fbec10553ec

**Comment by zhiyou1201** [comment]
5718里面你也没回复我这个问题呀

**Comment by pearmini** [comment]
看 scrollbar 相关的文档
